pom.xml

```diff
@@ -30,7 +30,7 @@
     <properties>
         <java.version>11</java.version>
         <kotlin.version>1.3.70</kotlin.version>
-        <fuel.version>2.2.1</fuel.version>
+        <fuel.version>2.3.1</fuel.version>
```

Comment:
her var det tydeligvis en breaking change. ser på saken

Reply:
kanskje det er mer hensiktsmessig å skrive bort fra fuel? Tror ikke jeg gidder å bruke så mye tid på å finne ut hvorfor det er så mye breakage mellom disse to versjonene 😅